test/integration/integration_kubernetes_test.go

```diff
@@ -416,3 +416,93 @@ kubernetes:
 	// https://github.com/kubernetes/kubernetes/pull/110479
 	asserter.droppedTargetCount(t, 1)
 }
+
+func Test_CuratedExperience(t *testing.T) {
+	t.Parallel()
+
+	k8sEnv := newK8sEnvironment(t)
+
+	// Create running pod
+	podBase := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "unmonitored-pod",
+			Labels: map[string]string{
+				"k8s.io/app": "myApp",
+			},
+			Annotations: map[string]string{
```

Comment:
I would remove this from the tests, and just scope it to discovery

Reply:
meaning, the goal of the test is to check if the curated experience targets are discovered and others are dropped but nothing more. (we have other tests for the annotations manipulation)